src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -337,9 +345,9 @@ public void keepHeading(double x, double y, boolean fieldRelative) {
   public int convertCardinalDirections(int povAngleDeg) {
     // change d-pad values for left and right to 20 degree angles
```

Comment:
If the angles are changing this comment should as well.

Reply:
Should probably just make the comment less specific since the exact angle is redundant